docs/tables/aws_cloudtrail_log/index.md

```diff
@@ -234,3 +261,12 @@ This table sets the following defaults for the [aws_s3_bucket source](https://hu
 | Argument      | Default |
 |---------------|---------|
 | file_layout   | `AWSLogs/(%{DATA:org_id}/)?%{NUMBER:account_id}/CloudTrail/%{DATA:region}/%{YEAR:year}/%{MONTHNUM:month}/%{MONTHDAY:day}/%{DATA}.json.gz` |
+
+### aws_cloudwatch_log_group
+
+This table sets the following defaults for the [aws_cloudwatch_log_group source](https://hub.tailpipe.io/plugins/turbot/aws/sources/aws_cloudwatch_log_group#arguments):
+
+| Argument      | Default |
+|---------------|---------|
+| log_group_name | `/aws/cloudtrail/logs` |
```

Comment:
Where are these defaults from? It feels odd to have defaults for both of these args.

Reply:
Also, are these actually defaults as defined by the table? I couldn't find them in the table files.

Reply:
Actually, we don't have any default value for any of the config args. Should we keep this section?